app/main.py

```diff
@@ -1,3 +1,41 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: float | int) -> object:
+        if isinstance(other, Distance):
+            other = other.km
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: float | int) -> object:
+        if isinstance(other, Distance):
+            other = other.km
+        self.km += other
+        return self
+
+    def __mul__(self, other: int | float) -> object:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> object:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: int | float) -> bool:
```

Comment:
The comparison methods below do not allow comparison between another Distance object